streamwebs_frontend/streamwebs/templates/streamwebs/site_detail.html

```diff
@@ -19,6 +19,16 @@
     var imageWidth = '200';
   </script>
 
+  <script>
+  $(document).ready(function(){
+    {% if messages %}
+      {% for message in messages %}
+      Materialize.toast('{{ message }}', 4000, 'red');
```

Comment:
If the message is a success, it should not be red. 

Also, we need to be consistent in our message display, if we use materialize toast, we need to make sure that is used everywhere in the application. If Django flash messages are used elsewhere, we should consider those here as well.